qmldir

```diff
@@ -1,3 +1,3 @@
 module Box2D
-plugin Box2D
+plugin qmlbox2d
```

Comment:
I had to update it for cmake else my project would not find the library.
It should cause issue for qbs, but I'm not familiar enough with it. I can see there is a Static and Dynamic libraries but with different names and none being Box2D...

Reply:
The Qbs project appears to call the library "box2d_lib" and does not actually build a QML plugin, so I think this line will not affect it. The Qbs example just links to that lib and then manually registers its types:

https://github.com/qml-box2d/qml-box2d/blob/2c20f757c37820a4c01611ee16d6ade86076666d/examples/main.cpp#L11-L13